### PR TITLE
Updated MDK tutorial based on feedback from customer

### PR DESCRIPTION
**Pull request by jitendrakansal**

### CODE REVIEW

File: tutorials/cp-mobile-dev-kit-map-extension/cp-mobile-dev-kit-map-extension.md [hunk 1]
```diff
@@ -460,9 +461,11 @@ Since you will display the customer's address in Google Maps on Android device,
 [DONE]
 [ACCORDION-END]
 
-[ACCORDION-BEGIN [Step 7: ](Create Your Branded MDK Client)]
+[ACCORDION-BEGIN [Step 7: ](Create Your Branded MDK Client (Required only for Android))]
 
-Follow steps 1 to 4 from [this tutorial](cp-mobile-dev-kit-build-client).
+For Android, you will need to pass the API key to the MDK client, there is no way public store client can access it, hence you will required to create a branded client using MDK SDK. Follow steps 1 to 4 from [this tutorial](cp-mobile-dev-kit-build-client).
```
Comment (MichaelCzcz) [inline tutorials/cp-mobile-dev-kit-map-extension/cp-mobile-dev-kit-map-extension.md:466]: Please correct grammar.

File: tutorials/cp-mobile-dev-kit-map-extension/cp-mobile-dev-kit-map-extension.md [hunk 2]
```diff
@@ -555,56 +558,50 @@ In this step, you will run the app on an android device.
 
 [OPTION BEGIN [iOS]]
 
-  1. In this step, you will run the app on an iOS device. Attach the device to your Mac and run `tns device ios` command to print a list of attached devices.
-
-    ![MDK](img_020.1.png)
+  1. SAP Web IDE has a feature to generate QR code for app on-boarding.
```
Comment (MichaelCzcz) [inline tutorials/cp-mobile-dev-kit-map-extension/cp-mobile-dev-kit-map-extension.md:561]: onboarding

File: tutorials/cp-mobile-dev-kit-map-extension/cp-mobile-dev-kit-map-extension.md [hunk 3]
```diff
@@ -555,56 +558,50 @@ In this step, you will run the app on an android device.
...
+    >**MDK Show QR Code** option is greyed out if MDK project is not yet deployed and activated as per step 5.
 
-    You can also run the app in Xcode. Open the project in Xcode with the command `open platforms/ios/<app name>.xcworkspace`, or open the workspace using the `File -> Open...` dialog in Xcode. Configure the application's code signing settings, then run the application for the target device.
+    ![MDK](img_012.1.png)
 
-    >To run the MDK client on iOS simulator, use `tns run ios --emulator` command.
+    >Once you have scanned and onboarded using the onboarding URL, it will be remembered. When you Log out and onboard again, same onboarding URL settings will be reused without the need to scan. You will need to use device Camera, if you would like to scan a different onboarding URL.
 
-  4. Once, above command gets successfully executed, you will see new MDK client up and running in your device.
+  2. Launch **`Mobile Svcs`** app on your iOS device. Tap **Scan** to start the device camera for scanning the on-boarding QR code.
```
Comment (MichaelCzcz) [inline tutorials/cp-mobile-dev-kit-map-extension/cp-mobile-dev-kit-map-extension.md:573]: onboarding

Comment (jitendrakansal) [reply tutorials/cp-mobile-dev-kit-map-extension/cp-mobile-dev-kit-map-extension.md]: updated

File: tutorials/cp-mobile-dev-kit-map-extension/cp-mobile-dev-kit-map-extension.md (hunk 1 above)
Comment (jitendrakansal) [reply tutorials/cp-mobile-dev-kit-map-extension/cp-mobile-dev-kit-map-extension.md]: updated

**Comment by jitendrakansal** [comment]
@MichaelCzcz Please have a look at this PR and merge if everything looks fine to you. 
Let me know if there is any change needed from my side.

**Pull request merged**